src/main/dialogs/captureToolDialog.js

```diff
@@ -0,0 +1,476 @@
+    var captureDirName = gpiiSettingsDir + "/capture-diagnostics-" + gpii.journal.formatTimestamp(startupTime);
+
+    fluid.log("Capture Diagnostics directory is: ", captureDirName);
+
+    var dialogPromise = electron.dialog.showMessageBox({
+        buttons: ["Cancel", "Run Capture"],
+        message: "Select 'Run Capture' to run the capture diagnostics and store them to disk.\n\n  After capture they will be stored in:\n" + captureDirName
+    });
+
+    // The promise returned from electrons prompt dialog will equal 1 if the user clicked ok,
+    // rather than cancel/close/etc.
+    // https://electronjs.org/docs/api/dialog#dialogshowmessageboxbrowserwindow-options
+    if (dialogPromise === 1) {
+        fs.mkdirSync(captureDirName);
+        gpii.app.captureTool.generateDiagnostics(diagnosticsCollector).then(
+            function (data) {
+                // 0. Solutions
+                fs.appendFileSync(captureDirName + "/solutions.json", JSON.stringify(data.solutions, null, 4));
+                // 1. Installed Solutions
+                fs.appendFileSync(captureDirName + "/installedSolutions.json", JSON.stringify(data.installedSolutions, null, 4));
+                // 2. Full System Settings Capture
+                fs.appendFileSync(captureDirName + "/settingsCapture.json", JSON.stringify(data.settingsCapture, null, 4));
+
+                electron.dialog.showMessageBox({
+                    message: "Capture diagnostics successfully taken and are now stored in this directory:\n" + captureDirName
+                });
+            },
+            function (err) {
+                electron.dialog.showMessageBox({
+                    message: "There was a problem capturing the diagnostics:\n" + err
+                });
+                fluid.log("Error running capture diagnostics. ", err);
+            }
+        );
+    }
+};
+
+/*
+ * Generates and returns a set of diagnostics for the current local machine.
+ *
+ * Returns a promise containing these.
+ */
+gpii.app.captureTool.generateDiagnostics = function (diagnosticsCollector) {
+    return fluid.promise.fireTransformEvent(diagnosticsCollector.events.onCollectDiagnostics, {});
+};
+
+/**
+ * Fetches the list of locally installed solutions and send them to the renderer
+ * on the `channelNotifier`.
+ *
+ * @param {gpii.app.channelNotifier} channelNotifier - The channel notifier to the renderer process.
+ * @param {gpii.flowManager.local} flowManager - The local flow manager.
+ */
+gpii.app.captureTool.channelGetInstalledSolutions = function (channelNotifier, flowManager) {
+    flowManager.capture.getInstalledSolutionsForCurrentDevice().then(
+        function (data) {
+            channelNotifier.events.sendingInstalledSolutions.fire(data);
+            var runningSolutions = {};
+            fluid.each(data, function (solutionEntry, solutionId) {
+                try {
+                    fluid.log("ABOUT TO CHECK PROCESS FOR: ", solutionId);
```

Comment:
If you mean to keep these log lines in, they should have a bit more of a clue that they originated from the capture tool

Reply:
Updated these to more informative and less dramatic (not all caps)